coding_guidelines_c++.md

```diff
@@ -694,36 +714,38 @@ Copy as `.clang-format` file into directory, in which you execute the `clang-for
     SpacesInParentheses: true
     SpacesInSquareBrackets: true
     Standard: Cpp03
-    
+
     UseTab: false
```

Comment:
Couldn't this block me formatted as anonymous code using just a three-backticks-block without a language specification?